08-authentication/03-sessions/app.js

```diff
@@ -32,40 +35,50 @@ app.use(async (ctx, next) => {
 app.use((ctx, next) => {
   ctx.login = async function(user) {
     const token = uuid();
+    const session = await Session.create({token, user, lastVisit: new Date()});
+
+    await session.save();
 
     return token;
   };
```

Comment:
I have 2 questions about this function:
1. Why do we need to have this function in `ctx`? I see no reason other than to show us that we can add methods to `ctx`.
2. Right now a user can create a lot of sessions. As I can see, this is normal behavior because he can use different devices, but we need to remove session when user logs out, right?